### PR TITLE
Restart app at the first connection

### PR DESCRIPTION
**Pull request by poupotte**

### CODE REVIEW

File: server/controllers/accounts.coffee [hunk 1]
```diff
@@ -12,6 +13,32 @@ errors = require '../middlewares/errors'
...
 user = new User()
 correctWitness = "Encryption is correct"
+apps = []
+
+restartApp = (app, cb) =>
+    homeClient = new Client 'http://localhost:9103'
+    homeClient.post "api/applications/#{app}/stop", {}, (err, res) ->
+        console.log err if err?
+        db.view 'application/byslug', {key: app}, (err, appli) ->
+            if appli[0]?
```
Comment (nono) [inline server/controllers/accounts.coffee]: @poupotte don't you think that an `else` clause is missing for calling cb?

**Comment by clochix** [comment]
This is a really important fix, @aenario or @frankrousseau could you review and merge it before we publish next version of the DS ?

File: server/controllers/accounts.coffee [hunk 2]
```diff
@@ -12,6 +13,34 @@ errors = require '../middlewares/errors'
...
+                    name: appli.slug
+                    domain: "127.0.0.1"
+                    repository:
+                        type: "git",
+                        url: appli.git
+                    scripts:
+                        start: "server.coffee"
+                    password: appli.password
+                homeClient.post "api/applications/#{app}/start", {start: descriptor}, (err, res) ->
+                    console.log err
```
Comment (frankrousseau) [inline server/controllers/accounts.coffee]: You should display err only if it exists.

**Comment by frankrousseau** [comment]
I merged too fast. The code is lacking of comments.

**Pull request merged**